### PR TITLE
Update donation links and fundraising form (Fixes #12628, #12698)

### LINKED ISSUE #12628: Update the "Donate" link in the footer to use our new donation platform
## Description

On the MoFo side, we're actively transitioning from donate.mozilla.org to a new donation platform named Fundraise Up. We'd like to request an update to the "Donate" footer link on mozilla.org to reflect that change and point users to the appropriate donation form.

mozilladonate

---

#### Success Criteria
- [ ] `href` of the footer link circled above is changed to `https://foundation.mozilla.org/?form=donate&utm_source=mozilla.org&utm_medium=referral&utm_content=company&utm_campaign=footer`

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

The Foundation's new donation form can now handle country / currency detection itself, so this PR simplifies both our `donate_url` helper and the fundraising form on the home page.

The MoFo team have mentioned that they may want to redesign the fundraising form in the future to make it a one button CTA, but for now this PR simply removes the previously offered presets from the current design, which are now mostly redundant.

## Issue / Bugzilla link

#12628
#12698

## Testing

http://localhost:8000/en-US/

- [ ] Donate links point to the new form along with utm params
- [ ] Fundraising banner submit redirects to the new form with utm params

### CODE REVIEW

File: bedrock/settings/base.py [hunk 1]
```diff
@@ -873,97 +873,7 @@ def get_app_name(hostname):
...
-DONATE_LINK = (
-    "https://donate.mozilla.org"
-    "?presets={presets}&amount={default}"
-    "&utm_source=mozilla.org&utm_medium=referral&utm_content={source}"
-    "&currency={currency}"
-)
-
-DONATE_LINK_UNKNOWN = "https://donate.mozilla.org/?utm_source=mozilla.org&utm_medium=referral&utm_content={source}"
-
-DONATE_PARAMS = {
```
[the remark this answers is not in the document]
Comment (maureenlholland) [reply bedrock/settings/base.py:885]: **chore**: clean the now-unused base helper `get_donate_params` https://github.com/mozilla/bedrock/blob/main/bedrock/base/templatetags/helpers.py#L156 & related test: https://github.com/mozilla/bedrock/blob/main/bedrock/base/tests/test_helpers.py#L74

File: bedrock/mozorg/templatetags/misc.py [hunk 2]
```diff
@@ -426,7 +426,7 @@ def press_blog_url(ctx):
 
 @library.global_function
 @jinja2.pass_context
-def donate_url(ctx, country="", source=""):
+def donate_url(ctx, campaign="", content=""):
     """Output a donation link to the donation page formatted using settings.DONATE_PARAMS
```
Comment (maureenlholland) [inline bedrock/mozorg/templatetags/misc.py:430]:
```suggestion
    """Output a formatted donation link to the donation page
```

**Review by maureenlholland — APPROVED** [review]
LGTM, much simpler

r+wc, some further clean-up needed with the deletion of settings.DONATE_PARAMS

File: bedrock/base/templates/includes/banners/fundraiser.html [hunk 3]
```diff
@@ -18,46 +18,18 @@
...
     </div>
 
-    <input type="hidden" name="currency" value="{{ donate_params.currency }}">
-    <input type="hidden" name="presets" value="{{ donate_params.presets }}">
-
     {# GA params #}
-    <input type="hidden" name="ref" value="giving2022banner">
-    <input type="hidden" name="utm_campaign" value="giving2022banner">
+    <input type="hidden" name="form" value="donate">
+    <input type="hidden" name="ref" value="home-page">
```
[the remark this answers is not in the document]
Comment (alexgibson) [reply bedrock/base/templates/includes/banners/fundraiser.html:28]: Great question. I've seen `ref` before as an example of a typical marketing attribution param, but I've no clue on how / when it was added here.

**Comment by alexgibson** [comment]
@maureenlholland thanks for the review! updated

**Review by maureenlholland — APPROVED** [review]
r++ 🟢

**Pull request merged**